userapi/src/main/java/fk/prof/userapi/api/AggregatedProfileLoader.java

```diff
@@ -28,6 +32,12 @@
  */
 public class AggregatedProfileLoader {
```

Comment:
It probably makes sense to have a well defined interface called AggeregatedProfileLoader to make the classes that depend on it, easier to test. The concrete impl (this one) can be called S3BackedProfileLoader

Reply:
Done.